test/suites/integration/bsp-volunteer.test.ts

```diff
@@ -211,7 +211,11 @@ for (const bspNetConfig of bspNetConfigCases) {
       // TODO: check the file key. We need an RPC endpoint to compute the file key.
 
       await it("downloaded file passed integrity checks", async () => {
-        await checkBspForFile("test/whatsup.jpg");
```

Comment:
Why not keeping this check as well?

Reply:
That check is just to wait for the file to be written and nothing more. After we get the response from the `saveFileToDisk` rpc, we know for sure the file is there (if success).